probes/hasOpenSSFBadge/def.yml

```diff
@@ -27,3 +27,9 @@ remediation:
     - Learn about best practices by following [the OpenSSF Best Practices Badge criteria](https://www.bestpractices.dev/en/criteria/0).
   markdown:
     - Learn about best practices by following [the OpenSSF Best Practices Badge criteria](https://www.bestpractices.dev/en/criteria/0).
+ecosystem:
+  languages:
+    - all
+  clients:
+    - cii_blob
+    - cii_http
```

Comment:
recommend dropping cii_blob and cii_http for reasons detailed above. Should be github and gitlab

Reply:
https://github.com/ossf/scorecard/pull/3797/commits/7bcf9773f5e16ed35f16f40534b74af860cc72f5